hello_vsi/FVP_Corstone_SSE-310.cfg

```diff
@@ -0,0 +1,6 @@
+# Parameters:
+# instance.parameter=value       #(type, mode) default = 'def value' : description : [min..max]
+#---------------------------------------------------------------------------------------------------
+mps3_board.visualisation.disable-visualisation=1      # (bool  , init-time) default = '0'          : Enable/disable visualisation
+mps3_board.DISABLE_GATING=1                           # (bool  , init-time) default = '0'          : Disable Memory gating logic

```

Comment:
mps3_board.DISABLE_GATING=1  can be removed, as default setting shall work as well

Reply:
Removed